native_client/ctcdecode/scorer.cpp

```diff
@@ -1,6 +1,35 @@
+#ifdef _MSC_VER
+#include <stdlib.h>
+#include <io.h> 
+#include <process.h> 
+#include <direct.h> 
+
+
+ 
+#define R_OK    4       /* Read permission.  */
+#define W_OK    2       /* Write permission.  */ 
+#define F_OK    0       /* Existence.  */
+
+#define access _access    
+#define ssize_t __int64
+#define STDIN_FILENO 0
+#define STDOUT_FILENO 1
+#define STDERR_FILENO 2 
+#endif
```

Comment:
Can you explain a bit those?

Reply:
We don't need this and two more includes, I'll remove them.